versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/MongoDbConstants.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.dremio.nessie.versioned.store.mongodb;
+
+/**
+ * Set of constants for the MongoDb versioned store.
+ */
+public class MongoDbConstants {
+  // The names of the collections in the database. These relate to
+  // {@link com.dremio.nessie.versioned.store.ValueType}
+  public static final String L1_COLLECTION = "l1";
+  public static final String L2_COLLECTION = "l2";
+  public static final String L3_COLLECTION = "l3";
+  public static final String REF_COLLECTION = "r";
+  public static final String VALUE_COLLECTION = "v";
```

Comment:
any good reason for short names here?

Reply:
These use the same names as the enumerates in ValueType.

Reply:
That doesn't seem like a good reason. The people managing Nessie are unlikely to be familiar with the internal Nessie code. In the dynamo case, we save these objects in a shared collection at the moment and used small names to avoid bloating every object more than necessary. It is an internal implementation detail as opposed to here where anyone who looks at Mongo is going to see these objects.

Reply:
Side question - do you foresee people working with the collections directly in Mongo, or is it simply for visibility as to what's stored and how?

Reply:
These will be updated to use:
nessie_l1
nessie_l2
nessie_l3
nessie_ref
nessie_value
nessie_key_fragment
nessie_commit_metadata